components/calendar/src/chinese_based.rs

```diff
@@ -208,10 +225,10 @@ impl<C: ChineseBased + CalendarArithmetic> ChineseBasedDateInner<C> {
     /// does not exceed the given fixed date. The day of month is calculated by subtracting the fixed date
     /// from the fixed date of the beginning of the month.
     ///
-    /// The calculation for `elapsed_years` in this function is based on code from _Calendrical Calculations_ by Reingold & Dershowitz.
+    /// The calculation for `elapsed_years` and `month` in this function are based on code from _Calendrical Calculations_ by Reingold & Dershowitz.
     /// Lisp reference code: https://github.com/EdReingold/calendar-code2/blob/main/calendar.l#L5414-L5459
     pub(crate) fn chinese_based_date_from_fixed(date: RataDie) -> Date<C> {
```

Comment:
This code does not look much like the Reingold code any more, but I trust that it still works